### PR TITLE
Calendar: Add the space between buttons(fixes 6796)

### PR DESCRIPTION
**Pull request by eonshikkim**
#6796 

The only changed I had made was adding  "margin-bottom: 15px;" after  "position: relative;"
White spaces were removed automatically when I pushed or saved it. I don't know why.

### CODE REVIEW

File: src/styles/calendar.scss [hunk 1]
```diff
@@ -1,40 +1,38 @@
 @import '~@fullcalendar/core/main.css';
 @import '~@fullcalendar/daygrid/main.css';
 @import '../app/variables';
-
```
Comment (paulbert) [inline src/styles/calendar.scss:4]: Please keep this line in the file.

File: src/styles/calendar.scss [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
-        content: "";
-        background-color: rgba(0, 0, 0, 0.04);
-        position: absolute;
-        top: 0;
-        left: 0;
-      }
-    }
-  }
-}
+}
```
Comment (paulbert) [inline src/styles/calendar.scss:38]: Please keep the newline at the end of the file.

**Comment by paulbert** [comment]
This messes up the alignment between the month and the buttons.  Need a better solution than just adding a margin.

**Comment by Mutugiii** [comment]
Stale, will need a new PR

**Pull request closed without merging**